tox.ini

```diff
@@ -1,23 +1,43 @@
 [tox]
-envlist = flake8,py{27,35,36}-django{18,19,110,111}
+envlist =
+    py{27,34,35,36}-dj111,
+    py{34,35,36}-dj20,
+    py{35,36}-djmaster,
+    flake8,
+    isort,
+    readme,
+    check-manifest
 skipsdist = True
 
 [testenv]
 deps =
-    django18: Django>=1.8,<1.9
-    django19: Django>=1.9,<1.10
-    django110: Django>=1.10,<1.11
-    django111: Django>=1.11,<1.12
-    -rtest-requires.txt
```

Comment:
If you don't think this line is not required, `test-requires.txt` file is not needed anymore.
And please explain your purpose.

Reply:
1. Depend libs are different for each python versions. 

    Like
    ```
    docutils<0.13  # 0.13 + py27 cause 'setup.py check' error. see: https://bugs.python.org/issue23063
    mock
    ```

2. Depends libs are different for each test (like flake8, isort, readme etc). This effective for test performance.

I made PR for test performance after merge this PR.